### PR TITLE
Added functions to run Express and Rails service wizards

### PR DESCRIPTION
**Pull request by msradam**
Added functions similar to the CreateDjangoService call that will run the docker-compose files from the Express and Rails service wizard subrepositories.

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -185,6 +185,42 @@ createDjangoService()
   )
 }
 
+# method to create Express services from scratch using Express wizard
+createExpressService()
```
Comment (jefmoura) [inline init.sh]: This is function isn't used anywhere. How/Where is it going to be called?

File: init.sh [hunk 2]
```diff
@@ -185,6 +185,42 @@ createDjangoService()
...
+  fi
+
+  (
+  cd "express-service-wizard" || return
+  docker-compose run --rm express_service_wizard || echo "Docker not configured, installed or running"
+  )
+}
+
+# method to create Ruby on Rails services from scratch using Rails wizard
+createRailsService()
```
Comment (jefmoura) [inline init.sh]: This is function isn't used anywhere. How/Where is it going to be called?

File: init.sh [hunk 3]
```diff
@@ -185,6 +185,42 @@ createDjangoService()
...
 }
 
+# method to create Express services from scratch using Express wizard
+createExpressService()
+{
+  # Check specific dependencies
+  type docker-compose >/dev/null 2>&1 || { echo >&2 "ERROR: You do not have 'Docker Compose' installed.
+  Check the documentation of how to install it: https://docs.docker.com/compose/install/"; exit 1; }
+
+  if [ ! -d express-service-wizard ]; then
```
Comment (jefmoura) [inline init.sh]: This folder doesn't exist, so the script always prints an error and exit

File: init.sh [hunk 4]
```diff
@@ -185,6 +185,42 @@ createDjangoService()
...
+}
+
+# method to create Ruby on Rails services from scratch using Rails wizard
+createRailsService()
+{
+  # Check specific dependencies
+  type docker-compose >/dev/null 2>&1 || { echo >&2 "ERROR: You do not have 'Docker Compose' installed.
+  Check the documentation of how to install it: https://docs.docker.com/compose/install/"; exit 1; }
+
+  if [ ! -d rails-service-wizard ]; then
```
Comment (jefmoura) [inline init.sh]: This folder doesn't exist, so the script always prints an error and exit

File: init.sh [hunk 5]
```diff
@@ -185,6 +185,42 @@ createDjangoService()
...
+  Check the documentation of how to install it: https://docs.docker.com/compose/install/"; exit 1; }
+
+  if [ ! -d express-service-wizard ]; then
+    MSG="The Express service wizard \"express-service-wizard\" wasn't found"
+    print_message "error" "$MSG"
+  fi
+
+  (
+  cd "express-service-wizard" || return
+  docker-compose run --rm express_service_wizard || echo "Docker not configured, installed or running"
```
Comment (jefmoura) [inline init.sh]: There's no `express_service_wizard`, so it always prints `Docker not configured, installed or running`

File: init.sh [hunk 6]
```diff
@@ -185,6 +185,42 @@ createDjangoService()
...
+  Check the documentation of how to install it: https://docs.docker.com/compose/install/"; exit 1; }
+
+  if [ ! -d rails-service-wizard ]; then
+    MSG="The Ruby on Rails service wizard \"rails-service-wizard\" wasn't found"
+    print_message "error" "$MSG"
+  fi
+
+  (
+  cd "rails-service-wizard" || return
+  docker-compose run web rails new YourRailsApplication --force --no-deps --database=postgresql || echo "Docker not configured, installed or running"
```
Comment (jefmoura) [inline init.sh]: There's no `YourRailsApplication`, so it always prints `Docker not configured, installed or running`

File: init.sh [hunk 7]
```diff
@@ -389,15 +390,6 @@ deployBuildlyCore()
...
     --set gcp.cloudsql.secretName="$cloudsql_secret"
-  elif [[ -n "$1" && ("$1" == "minikube" || "$1" == "Minikube") ]] ;then
-    helm install buildly-core . --namespace buildly \
-    --set configmap.data.DATABASE_HOST="$dbhost" \
-    --set configmap.data.DATABASE_PORT=\""$dbport"\" \
-    --set secret.data.DATABASE_USER="$dbuser" \
-    --set secret.data.DATABASE_PASSWORD="$dbpass" \
-    --set buildly.image.repository=buildly-core \
-    --set buildly.image.version=latest \
-    --set buildly.image.pullPolicy=IfNotPresent
```
Comment (jefmoura) [inline init.sh]: This code block doesn't need to be removed.

File: init.sh [hunk 8]
```diff
@@ -413,13 +405,11 @@ deploy2Minikube()
...
-  setupServices "buildly"
-  setupServices
+  # deploy buildly using helm charts
+  deployBuildlyCore
 
-  # deploy buildly and services to a minikube instance
-  deployBuildlyCore "minikube"
-  deployServices
+  # build local images for each service
+  setupServices
```
Comment (jefmoura) [inline init.sh]: This code block doesn't need to be changed to add new servicer wizards

File: init.sh [hunk 9]
```diff
@@ -632,13 +622,6 @@ cat <<EOF
 
 ${BOLD}${WHITE}Buildly CLI 0.0.1${OFF}
 
-If it's your first time using this tool, you probably want to create an application,
-so you can just execute this script with the option --create-application or -ca, e.g,
-
-'''
-$script_name --create-application
-'''
-
```
Comment (jefmoura) [inline init.sh]: We need a good message when the help option is used

File: init.sh [hunk 10]
```diff
@@ -783,9 +766,8 @@ esac
 done
 
 if [[ -z "$action" ]]; then
-  MSG="Usage: $script_name [OPTION]\nTry '$script_name --help' for more information."
-  echo -e "$MSG"
-  exit 0
+  MSG="No action specified!"
+  print_message "error" "$MSG"
```
Comment (jefmoura) [inline init.sh]: We need to print a helpful message and stop the script when no option is used as some Unix commands already do.

**Review by jefmoura — CHANGES_REQUESTED** [review]

**Review by glind — APPROVED** [review]

**Comment by jefmoura** [comment]
Currently, we don't have new and ready to be used application wizards, so we'll close this pull request and this functionality will be implemented in the near future when we have new wizards available.

**Pull request closed without merging**